src/CMakeLists.txt

```diff
@@ -2,8 +2,50 @@ cmake_minimum_required(VERSION 3.22)
 
 set(CMAKE_CXX_STANDARD 20)
 set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(CMAKE_BUILD_TYPE RelWithDebInfo)
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS_DEBUG} -g")
```

Comment:
It should be `set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -g")`

Reply:
Done